src/index.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import P5Wrapper from "./index";
+import sketch from "./__fixtures__/sketch";
+
+describe("P5Wrapper", () => {
+  xit("renders the component", () => {
```

Comment:
@jamesrweb I don't know how to properly render the canvas component in the test and what to assert here. Could you help with that?

Reply:
@yevdyko added an initial render test, can you review and check why CI is failing?

Reply:
@jamesrweb Thanks for your input!

I'm still looking into the problem in CI. Something is wrong with the link to the forked branch. I need to double-check one solution in the test repository before asking for changes on your side.

Reply:
@jamesrweb After playing around with the test repo a bit and diving into the `git-auto-commit-action` documentation, I found that it doesn't work well with the forked repository due to Github actions restrictions. See the following details: [Using the Action in forks from public repositories](https://github.com/stefanzweifel/git-auto-commit-action#using-the-action-in-forks-from-public-repositories).

So I would prefer to remove auto-correct steps from CI and show it failing if the changes don't pass the formatter or linter. What do you think about that?

Reply:
Sure but please reset the `ref` param on all instances to the `head_ref` then. Also we could create a separate workflow which only works when new code is pushed to master from local or a PR which does the formatting and fixes. Then the PR workflow just does linting and testing for example. What do you think?

Reply:
Totally agreed, I've updated the current workflow based on your suggestions. This way we can use one workflow, but only run the formatting and auto-correct steps on the push event. I think we can stop there for now and continue to improve the flow in the next iterations.

🚢 🚢